src/Zap.sol

```diff
@@ -485,7 +552,7 @@ contract Zap is Errors {
     /// @dev allowance is assumed
     /// @dev following execution, this contract will hold any excess USDx
     function _burn(uint256 _amount, uint128 _accountId) internal {
```

Comment:
When calling `burn`externally, if debt is repaid without overpaying, `remaining`will be `0`and `_push(USDX, msg.sender, remaining)` will revert as USDX does not support 0 amount transfers.

Need to check that `remaining > 0` before pushing back any excess USDx to the caller.

Reply:
fixed good catch 🫡